packages/scripts/refactor/entry-refactor.ts

```diff
@@ -20,7 +20,8 @@ async function entryRefactor() {
         .get()
         .then((snapshot) => {
           snapshot.forEach((dictionary) => {
-            console.log('--------------------Refactoring: ', dictionary.id);
+            console.log('--------------------Refactoring: ');
+            console.log(dictionary.id);
```

Comment:
@jacob-8 I'm breaking the logs because for some reason my console don't show them in one line